libcxx/include/locale

```diff
@@ -366,9 +366,11 @@ _LIBCPP_HIDE_FROM_ABI _ForwardIterator __scan_keyword(
 
 struct _LIBCPP_EXPORTED_FROM_ABI __num_get_base {
   static const int __num_get_buf_sz = 40;
+  static const size_t __int_chr_cnt = 26;
+  static const size_t __fp_chr_cnt  = 28;
```

Comment:
Thanks for removing magic numbers, can you add some comment what these values are use for?

Reply:
Added.